atomic_reactor/util.py

```diff
@@ -1520,30 +1523,41 @@ def get_log_files(self, osbs, pipeline_run_name):
         """
 
         logs = None
-        output = []
+        outputs = []
 
         # Collect logs from server
         try:
             logs = osbs.get_build_logs(pipeline_run_name)
         except OsbsException as ex:
             self.log.error("unable to get build logs: %s", ex)
-            return output
+            return outputs
 
-        # OSBS2 TBD logs for specific arches for binary builds
         filename = 'osbs-build'
-        logfile = NamedTemporaryFile(prefix=f"{pipeline_run_name}-{filename}",
-                                     suffix=".log", mode='wb')
+        logfiles = {'noarch': NamedTemporaryFile(prefix=f'{pipeline_run_name}-{filename}-',
```

Comment:
missing 'noarch' at the end of prefix

Reply:
I had removed it because we were adding final name in metadata and temp files are named different but added it back.